semver.js

```diff
@@ -1186,3 +1186,9 @@ function outside(version, range, hilo, loose) {
   }
   return true;
 }
+
+exports.prerelease = prerelease;
+function prerelease(version) {
```

Comment:
should take a second "loose" boolean arg, and pass that to the new Semver in the next line.